src/main/java/org/apereo/openlrs/controllers/caliper/CaliperController.java

```diff
@@ -52,7 +58,46 @@
     private CaliperService caliperService;
     @Value("${caliper.version}")
     private String version;
+    @Value("${caliper.pageDefault ?: 0}")
+    private Integer pageDefault;
+    @Value("${caliper.limitDefault ?: 20}")
```

Comment:
I guess I do not understand what is default limit vs max limit. Limit is a limit, and it should be fixed...